docs/experiments.rst

```diff
@@ -0,0 +1,318 @@
+.. _Experiments:
+
+Experiments
+===========
+
+You might want to take a look at the following pages before exploring experiments:
+
+- :ref:`QuickStart`
+- :ref:`AtVariables`
+- :ref:`Instances`
+- :ref:`Builds`
+- :ref:`Revisions`
+- :ref:`Variants`
+
+On this page we describe how to specify experiments in our ``experiments.yml`` file. We will see how
+to use builds in experiments and enable options such as repeating experiments or setting a timer.
+Moreover, we will see how to modify the ``sbatch`` command when using simexpal with Slurm and how to
+set environment variables.
+
+Specifying Experiments
+----------------------
+
+To specify an experiment we use the keys
+
+- ``name``: name of the experiment
+- ``args``: list of experiment arguments
+- ``output``: where experiment outputs should be stored.
+
+For now, choose ``stdout`` as value for the ``output`` key. Simexpal will then store the outputs
+of the experiments in the following directory:
+
+- ``/experiments.yml_directory/output/<experiment_name>~<variant_names>@<revision_name>``
```

Comment:
`experiments.yml_directory` looks a bit awkward. Can we just state relative paths?

Reply:
Done. As PR #69 was merged, I created a new section for the ``output``/``stdout`` key (the section can be found below the "Use Builds" section) and added the ``@OUTPUT:<ext>@`` to the at-variables.